l10n_it_fatturapa/README.rst

```diff
@@ -109,6 +109,7 @@ Contributors
 * Roberto Onnis
 * Alessio Gerace
 * Sergio Zanchetta <https://github.com/primes2h>
+* Gianluigi Tiesi <https://github.com/sherpya>
```

Comment:
https://github.com/OCA/l10n-italy/pull/1243#issuecomment-495158305

Reply:
Aggiunto

Reply:
thanks ma poi non lo cancella il bot se non è in CONTRIBUTORS?

Reply:
@sherpya, come puoi verificare, @eLBati ora l'ha aggiunto anche a `CONTRIBUTORS`.